doubly_linked_list/doubly_linked_list.py

```diff
@@ -97,7 +97,13 @@ def move_to_front(self, node):
     self.length = self.length -1
 
   def move_to_end(self, node):
-    pass
+    if node == self.tail:
+      return None
+    self.add_to_tail(node.value)
+    if node == self.head:
```

Comment:
If the node that is moved to the end of the DLL is not the head and it is located somewhere in the middle of the DLL, you also need to delete it at its initial position because the previous and next pointers of adjacent nodes when applicable will not be correctly modified. This is scenario that was not added as a unit test for this sprint, but it is important to think about when coding this method.